Added info how to update themes to Obsidian 0.13+ (#172)
obsidian-community/obsidian-hub: 04 - Guides, Workflows, & Courses/Guides/How to update your plugins and CSS for live preview.md

## For Theme Developers

- `.CodeMirror-line` has changed to `.cm-line`
- active line has a new selector (`.cm-active`), that needs to be added for active line highlighting
- Cursor styling has a new selector (`.cm-s-obsidian .cm-cursor`)
- fold markers (the "..." when a a heading / list item is folded) has a new selector (`.cm-foldPlaceholder`)
- the gutter has a new selector (`.markdown-source-view.mod-cm6 .cm-gutters`)

> - the css class for the search/replace modal (`.document-search-container`) is does not have `position: relative` by default anymore, so you need to add that when the theme moves the location of that modal

Reviewer: ```suggestion
- the css class for the search/replace modal (`.document-search-container`) does not have `position: relative` by default anymore, so you need to add that when the theme moves the location of that modal
```

- Collapse indicators in Edit Mode are not part of the Code mirror line anymore, meaning they cannot be targeted individually anymore.

> - Live preview being a hybrid of edit and preview mode also also shares certain css classes with *both*, meaning some css targeting elements in preview/edit mode will probably be unintentionally target live preview as well. 

Reviewer: ```suggestion
- Live preview being a hybrid of edit and reading mode also shares certain css classes with *both*, meaning some css targeting elements in reading/edit mode will probably unintentionally target live preview as well. 
```

> 	- Example: `.external` is used to style external link in Preview Mode, but is now used in Live preview as well. 

Reviewer: ```suggestion
	- Example: `.external` is used to style external links in Reading Mode, but is now used in Live preview as well. 
```

> 	- Mostly, these issues can be solved via more specific selectors, e.g. `a.external` for Preview Mode and `span.external` for Edit Mode

Reviewer: ```suggestion
	- Mostly, these issues can be solved via more specific selectors, e.g. `a.external` for Reading Mode and `span.external` for Edit Mode
```

> - trailing whitespace have a CSS class now, allowing them to target them, e.g. 

Reviewer: ```suggestion
- trailing whitespace have a CSS class now, allowing to target them, e.g. 
```
